frontend/src/components/auth/Signup/index.tsx

```diff
@@ -28,13 +31,23 @@ const Signup = () => {
     password: "",
     confirmPassword: "",
     businessName: "",
-    role: Role.DONOR,
+    role: "",
+    acceptedTerms: false,
   });
 
   const { step, navigation }: UseStepType = useStep({ steps, initialStep: 0 });
   const { id } = step;
+  const { next } = navigation;
 
   switch (id) {
+    case "account type":
```

Comment:
not sure if this is the best way to handle "skipping" the terms and conditions page for donors... any suggestions/feedbakc is appreciated!